### PR TITLE
Change all.bash to run all tests again with -txn flag

### PR DESCRIPTION
**Pull request by mohit10verma**

### CODE REVIEW

File: src/all.bash [hunk 1]
```diff
@@ -11,5 +11,8 @@ fi
 OLDPATH="$PATH"
 . ./make.bash "$@" --no-banner
 bash run.bash --no-rebuild
+export GO_GCFLAGS="-txn"
```
Comment (jerrinsg) [inline src/all.bash:14]: is `GO_GCFLAGS` used to specify any other flags? Is writing this line as  `export GO_GCFLAGS="$GO_GCFLAGS -txn"` safer?

**Comment by jerrinsg** [comment]
Also, a lot of the output log in the test run says `(cached)` in the travis log (https://travis-ci.com/jerrinsg/go-pmem/builds/124795350) and it looks like those tests were not re-run the second time. Is this okay or should we do a force re-run of the tests?

**Comment by mohit10verma** [comment]
> Also, a lot of the output log in the test run says `(cached)` in the travis log (https://travis-ci.com/jerrinsg/go-pmem/builds/124795350) and it looks like those tests were not re-run the second time. Is this okay or should we do a force re-run of the tests?

Yeah, Ideally we should force the re-run for the second time. Do you know how to do that?

**Comment by jerrinsg** [comment]
One option is to call `clean.bash` and run `all.bash` again with the required flags set. I am not sure if there is a better way to force re-run the tests.
Also, instead of modifying `all.bash`, we can alternatively automate this workflow by adding these scripts in `travis.yml`.

**Comment by mohit10verma** [comment]
Closing this. Sorry for all the confusion. I'll open a new pull request that **ONLY** changes the all.bash script.

**Pull request closed without merging**